### PR TITLE
Add OpenCensus tracing to proxy config

### PR DESCRIPTION
**Pull request by objectiser**
Envoy has recently added support for the OpenCensus tracer. This PR adds the configuration to support that tracer.

Support for OpenCensus tracer has been listed as an item on the Policy/Telemetry roadmap for 1.3.

### CODE REVIEW

File: mesh/v1alpha1/proxy.proto [hunk 1]
```diff
@@ -66,6 +66,48 @@ message Tracing {
     string address = 1;
   }
 
+  // OpenCensus defines configuration for a OpenCensus tracer.
```
Comment (kyessenov) [inline mesh/v1alpha1/proxy.proto]: Would it be better to import xDS/opencensus proto instead?

**Comment by mandarjog** [comment]
https://github.com/istio/api/pull/959
@objectiser please see this .

This is too much details leaking into the mesh config api.
We are going to have better ways to patch xds defaults.

Also I was wondering would we want to expose opencensus config or the actual 
Exporter it enables?

In the stack driver case I have chosen to indicate stack driver.

File: mesh/v1alpha1/proxy.proto (hunk 1 above)
Comment (objectiser) [reply mesh/v1alpha1/proxy.proto]: Sounds reasonable although why did the other tracer configs not use that approach?

**Comment by objectiser** [comment]
Closing in favour of #959

**Pull request closed without merging**